RecoLocalTracker/SiPixelClusterizer/plugins/gpuCalibPixel.h

```diff
@@ -96,18 +98,18 @@ namespace gpuCalibPixel {
       if (invalidModuleId == id[i])
         continue;
 
-      constexpr int mode = (Phase2ReadoutMode < -1 ? -1 : Phase2ReadoutMode);
+      const int mode = (Phase2ReadoutMode < -1 ? -1 : Phase2ReadoutMode);
```

Comment:
why these are no longer `constexpr` ?

Reply:
Because now `SiPixelClusterThresholds` may come from configuration (I moved it to be configurable since it happened some of those changed in the past and went unnoticed being hardcoded).

Reply:
Understood.